pages/common/gnuplot.md

```diff
@@ -0,0 +1,19 @@
+# gnuplot
+
+> Optionally interactive graph plotter that outputs in several formats.
+
+- Start the interactive graph plotting shell:
+
+`gnuplot`
+
+- Plot the graph for the specified graph definition file:
+
+`gnuplot {{path/to/definition.plt}}`
+
+- Set the output format by executing a command before loading the definition file (in this case output a PNG to stdout):
```

Comment:
"output a PNG to stdout" means dumping the binary file contents to stdout. That is probably
 not what is happening. Maybe better to say "redirect output to a png file".

Reply:
Hrm, yeah. I had trouble with this, because whether it outputs to stdout or not depends on the graph definition file. For example, I could say this:

```
set terminal png size 1024,768
```

...which would output the png to the standard output, or this:

```
set output "filename.png"
```

....which would output to a file directly..

Reply:
Maybe just go with the most common usage and mention `set output "filename.png"` ? No need to cover all scenarios and make the description long in that process

Reply:
True. Interestingly enough, it's actually `set terminal png` that you see all over stackoverflow - even though it's not as useful as `set output`. Still,  agree - I'll change it.